### PR TITLE
small-code-quality-improvements

### PR DESCRIPTION
**Pull request by TheRealHaui**
Made small code quality improvements.

### CODE REVIEW

File: src/main/java/org/apache/commons/text/FormattableUtils.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
      * <p>This constructor is public to permit tools that require a JavaBean
      * instance to operate.</p>
      */
+    @SuppressWarnings("squid:S1118")
```
Comment (kinow) [inline src/main/java/org/apache/commons/text/FormattableUtils.java]: Is that a suppress warning for SonarQube? I am not sure if others are actively looking at the reports produced by SonarQube. I think this is not really necessary, if indeed just for SonarQube.

File: src/main/java/org/apache/commons/text/WordUtils.java [hunk 2]
```diff
@@ -614,13 +615,11 @@ public static String swapCase(final String str) {
         for (int index = 0; index < strLen;) {
             final int oldCodepoint = str.codePointAt(index);
             final int newCodePoint;
-            if (Character.isUpperCase(oldCodepoint)) {
+            if (Character.isUpperCase(oldCodepoint) || Character.isTitleCase(oldCodepoint) ) {
```
Comment (kinow) [inline src/main/java/org/apache/commons/text/WordUtils.java]: Nice catch. Any chance you could add a unit test for that too?

File: src/main/java/org/apache/commons/text/WordUtils.java [hunk 3]
```diff
@@ -614,13 +615,11 @@ public static String swapCase(final String str) {
...
                 newCodePoint = Character.toLowerCase(oldCodepoint);
                 whitespace = false;
-            } else if (Character.isTitleCase(oldCodepoint)) {
-                newCodePoint = Character.toLowerCase(oldCodepoint);
-                whitespace = false;
-            } else if (Character.isLowerCase(oldCodepoint)) {
+            }
+            else if (Character.isLowerCase(oldCodepoint)) {
```
Comment (kinow) [inline src/main/java/org/apache/commons/text/WordUtils.java]: Any reason for breaking the line here? Not sure if it's just GitHub UI getting confused, or if the bracket is not in the same line as the else if statement...

File: src/main/java/org/apache/commons/text/FormattableUtils.java (hunk 1 above)
Comment (TheRealHaui) [reply src/main/java/org/apache/commons/text/FormattableUtils.java]: Well, works with Sonaqube and Intellij IDEA.
Therefore I assume Eclipse and Netbeans recognizing it too.

I guess you want it to removed.
True?

File: src/main/java/org/apache/commons/text/WordUtils.java (hunk 2 above)
Comment (TheRealHaui) [reply src/main/java/org/apache/commons/text/WordUtils.java]: The corresponding lines as the whole method are already completely covered by existing Unit Tests.
selection_050

File: src/main/java/org/apache/commons/text/WordUtils.java (hunk 3 above)
Comment (TheRealHaui) [reply src/main/java/org/apache/commons/text/WordUtils.java]: No, you're right.
My mistake.

Corrected and commited the correction.

File: src/main/java/org/apache/commons/text/FormattableUtils.java (hunk 1 above)
Comment (PascalSchumacher) [reply src/main/java/org/apache/commons/text/FormattableUtils.java]: -1 to this

Eclipse was not displaying any warning before, but with this change it displays this warning: `Unsupported @SuppressWarnings("squid:S1118")`

Comment (TheRealHaui) [reply src/main/java/org/apache/commons/text/FormattableUtils.java]: Removed SuppressWarnings anootations completely.

File: src/main/java/org/apache/commons/text/WordUtils.java [hunk 4]
```diff
@@ -614,10 +614,7 @@ public static String swapCase(final String str) {
         for (int index = 0; index < strLen;) {
             final int oldCodepoint = str.codePointAt(index);
             final int newCodePoint;
-            if (Character.isUpperCase(oldCodepoint)) {
-                newCodePoint = Character.toLowerCase(oldCodepoint);
-                whitespace = false;
-            } else if (Character.isTitleCase(oldCodepoint)) {
+            if (Character.isUpperCase(oldCodepoint) || Character.isTitleCase(oldCodepoint) ) {
```
Comment (chtompki) [inline src/main/java/org/apache/commons/text/WordUtils.java]: Checkstyle want's to see this as

```java
if (Character.isUpperCase(oldCodepoint) || Character.isTitleCase(oldCodepoint)) {
```

Comment (TheRealHaui) [reply src/main/java/org/apache/commons/text/WordUtils.java]: Removed white space.

**Comment by chtompki** [comment]
That last build failure wasn't on you....just fixed it in the master branch.

**Comment by TheRealHaui** [comment]
@chtompki 
If you want me to re-push any minor change to re-trigger the build just tell.

**Comment by TheRealHaui** [comment]
Well, of course!
And much, much, much more in Java!

**Comment by TheRealHaui** [comment]
:-)

**Comment by PascalSchumacher** [comment]
Merged in : https://github.com/apache/commons-text/commit/5598109efb2365384b9a9110323ed9a7004cc905 Thanks!

**Comment by TheRealHaui** [comment]
You're welcome.

**Pull request closed without merging**